sdk/cosmosdb/cosmos/src/utils/encode.ts

```diff
@@ -0,0 +1,7 @@
+export function encodeUTF8(str: string): Uint8Array {
```

Comment:
In the future we may want to extract this function to @azure/core-crypto since this [overlaps with keyvault](https://github.com/Azure/azure-sdk-for-js/blob/e44ecb31a76dd1ea636cd776cafac3addc5046b9/sdk/keyvault/keyvault-keys/test/utils/crypto.ts#L10)